common-rest/src/main/java/org/uniprot/api/rest/output/converter/ErrorMessageXMLConverter.java

```diff
@@ -4,27 +4,26 @@
 import java.io.OutputStream;
 import java.lang.reflect.Type;
 
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+
 import org.springframework.http.HttpInputMessage;
 import org.springframework.http.HttpOutputMessage;
 import org.springframework.http.MediaType;
 import org.springframework.http.converter.AbstractGenericHttpMessageConverter;
 import org.springframework.http.converter.HttpMessageNotReadableException;
 import org.springframework.http.converter.HttpMessageNotWritableException;
 import org.springframework.lang.Nullable;
-import org.uniprot.api.rest.output.UniProtMediaType;
 import org.uniprot.api.rest.validation.error.ErrorInfo;
 
-import jakarta.xml.bind.JAXBContext;
-import jakarta.xml.bind.JAXBException;
-import jakarta.xml.bind.Marshaller;
-
 /** @author lgonzales */
 public class ErrorMessageXMLConverter extends AbstractGenericHttpMessageConverter<ErrorInfo> {
 
     private final Marshaller marshaller;
 
     public ErrorMessageXMLConverter() {
-        super(MediaType.APPLICATION_XML, UniProtMediaType.RDF_MEDIA_TYPE);
+        super(MediaType.APPLICATION_XML);
```

Comment:
For RDF, why do you think we should remove it?

Reply:
Added it back